mm2src/rpc/lp_commands/lp_commands.rs

```diff
@@ -4,8 +4,11 @@ use common::HttpStatusCode;
 use crypto::{CryptoCtx, CryptoInitError};
 use derive_more::Display;
 use http::StatusCode;
+use rpc::v1::types::H160;
```

Comment:
Could you please use `H160Json` alias for it like here
https://github.com/KomodoPlatform/atomicDEX-API/blob/778461afccf42c17da0e02394ef6890a092a7a99/mm2src/lp_swap/recreate_swap_data.rs#L12